src/main/scala/de/upb/cs/uc4/hyperledger/utilities/helper/TransactionHelper.scala

```diff
@@ -236,12 +183,12 @@ protected[hyperledger] object TransactionHelper {
       transactionPayload: ByteString
   ): CompletableFuture[BlockEvent#TransactionEvent] = {
     try {
-      val orderers = if (ReflectionHelper.getPrivateField(transactionOptions)("orderers")() != null) ReflectionHelper.getPrivateField(transactionOptions)("orderers")().asInstanceOf[util.List[Orderer]]
-      else new util.ArrayList[Orderer](channel.getOrderers())
+      val orderers = if (ReflectionHelper.getPrivateField(transactionOptions)("orderers") != null) ReflectionHelper.getPrivateField(transactionOptions)("orderers").asInstanceOf[util.List[Orderer]]
+      else new util.ArrayList[Orderer](channel.getOrderers)
       val shuffeledOrderers: util.ArrayList[Orderer] = new util.ArrayList[Orderer](orderers)
-      if (ReflectionHelper.getPrivateField(transactionOptions)("shuffleOrders")().asInstanceOf[Boolean]) Collections.shuffle(shuffeledOrderers)
+      if (ReflectionHelper.getPrivateField(transactionOptions)("shuffleOrders").asInstanceOf[Boolean]) Collections.shuffle(shuffeledOrderers)

```

Comment:
rename string to shuffleOrderers

Reply:
I think shuffelOrders is the correct Field, is it not?

Reply:
Just a minor thing, the field is called shuffledOrderers, so plus "er" in the end, the string is missing that

Reply:
```java
/**
     * TransactionOptions class can be used to change how the SDK processes the Transaction.
     */
    public static class TransactionOptions {
        List<Orderer> orderers;
        boolean shuffleOrders = true;
        NOfEvents nOfEvents;
        User userContext;
        boolean failFast = true;

....


        public TransactionOptions shuffleOrders(boolean shuffleOrders) {
            this.shuffleOrders = shuffleOrders;
            return this;
        }
```
this is the transactionOptions class of the transactionOptions-object we want to access the field "shuffelOrders" not the method "shuffleOrderers"